scripts.js

```diff
@@ -81,8 +93,23 @@ function createSection(colors) {
     asideSection.appendChild(newElement);
   }
   newElement.innerHTML += `
-    <button><span class="material-symbols-outlined"> delete </span></button>
-    `;
+  <button class="delete-button"><span class="material-symbols-outlined" id="${paletteId}"> delete </span></button>
+  `;
+
 
   createNewPalette();
 }
+
+// Delete Palette 
+
+function deletePalette(event){
+  event.preventDefault();
+  for (var i = 0; i < savedPalettes.length; i++) {
+    if (savedPalettes[i].id == event.target.id) {
+      savedPalettes.splice(i, 1);
+    };
+  };
+  
+  event.target.parentNode.parentNode.remove()
+  }
+
```

Comment:
This looks great! On line 106, I tested with and without and couldn't tell what event default is being prevented. Making a note here to remind me for our check-in tomorrow!